kollider/docker-compose.yml

```diff
@@ -18,9 +18,9 @@ services:
     environment:
       LND_IP: $APP_LIGHTNING_NODE_IP
       LND_ZMQ_SUB_ADDRESS: "tcp://kollider_ws_1:5556"
-  
+
   ws:
-    image: kolliderhq/kollider-ws-client:v1.0.6@sha256:05f3d1ddf9949b377788e442a1050457559291a50510f5d6dfa66a28c283a80a
+    image: kolliderhq/kollider-ws-client:v1.0.7@sha256:7136a6b25b452c93ed2762f5ab00e6a6d9a76d586c1eb1bdf962bee684135cd5
```

Comment:
It looks like this image is not built for `arm64` (https://hub.docker.com/layers/kolliderhq/kollider-ws-client/v1.0.7/images/sha256-7136a6b25b452c93ed2762f5ab00e6a6d9a76d586c1eb1bdf962bee684135cd5?context=explore).

I am getting this error:
```
[FATAL tini (7)] exec docker-entrypoint.sh failed: Exec format error
[FATAL tini (7)] exec docker-entrypoint.sh failed: Exec format error
[FATAL tini (6)] exec docker-entrypoint.sh failed: Exec format error
[FATAL tini (6)] exec docker-entrypoint.sh failed: Exec format error
```